pkg/manifest/manifest_writer.go

```diff
@@ -23,12 +23,12 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
-type ManifestWriterContext struct {
+type WriterContext struct {
```

Comment:
Advanced commit nitpick: You don't actually document anything in this commit ;) I guess the msg is copied from the LoaderContext commit

Reply:
Thanks, the commit was missing code :)